xline/tests/common/mod.rs

```diff
@@ -112,6 +126,12 @@ impl Cluster {
     }
 }
 
+impl Drop for Cluster {
+    fn drop(&mut self) {
+        self.stop();
```

Comment:
I think we can inline `stop` method here.

Reply:
fixed